exonum/src/helpers/mod.rs

```diff
@@ -17,7 +17,6 @@
 pub use self::types::{Height, Milliseconds, Round, ValidatorId, ZeroizeOnDrop};
 
 pub mod config;
-pub mod fabric;
```

Comment:
Module files will be deleted once we verify that all the needed functionality is transfered.